lib/services/android-project-service.ts

```diff
@@ -289,8 +289,8 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 
 		this.$fs.ensureDirectoryExists(platformsAppResourcesPath);
 
-		this.$fs.copyFile(path.join(projectAppResourcesPath, platformData.normalizedPlatformName, "*"), platformsAppResourcesPath);
-			const appResourcesDirStructureHasMigrated = this.$androidResourcesMigrationService.hasMigrated(projectAppResourcesPath);
+		this.$fs.copyFile(path.join(projectAppResourcesPath, platformData.normalizedPlatformName, "*"), path.dirname(platformsAppResourcesPath));
```

Comment:
maybe I'm mistaken, but isn't the whole row on a wrong place? Currently, in case the Android App_Resources are migrated, we will copy them here and on line 295.
I think the correct fix is to move the current line in the else statement below.